<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ngw9-PC" ignored="0" name="Test" tests="2" failures="1" timestamp="01 三月 2018 10:18:08 GMT" time="34.773" errors="0">
  <testcase name="loginpasserror" time="10.429" classname="com.niuguwang.cases.casetest">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0at java.util.ArrayList.rangeCheck(Unknown Source)at java.util.ArrayList.get(Unknown Source)at com.niuguwang.base.OperateAppium.inputManyText(OperateAppium.java:312)at com.niuguwang.operation.LoginTest.login(LoginTest.java:31)at com.niuguwang.cases.casetest.loginpasserror(casetest.java:27)... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- loginpasserror -->
  <testcase name="login" time="24.305" classname="com.niuguwang.cases.casetest"/>
</testsuite> <!-- Test -->
